plugins/arsenal.py

```diff
@@ -21,35 +23,43 @@ async def banager(message: Message, chat_id: int, user_id: int, until_date: int)
         return await banager(message, chat_id, user_id, until_date)
     except Exception as u_e:
         if hasattr(u_e, 'NAME'):
-            log_msg = f'ERROR:- {u_e.NAME} >> {type(u_e).__name__} > {u_e.MESSAGE}'
+            log_msg = (
+                f"ERROR:- {u_e.NAME} >>"
+                f" {type(u_e).__name__} > {u_e.MESSAGE}"
+            )
         else:
             log_msg = f'ERROR:- {type(u_e).__name__} > {str(u_e)}'
     return log_msg
 
 
 @userge.on_cmd("snap", about={
     'header': "Ban All",
-    'description': "Haha, a Mighty Thanos snap to Ban All Members of a SuperGroup",
+    'description': "Haha, a Mighty Thanos snap to Ban"
+    " All Members of a SuperGroup",
```

Comment:
is this indent correct ?

Reply:
either don't break line or give it true indent

Reply:
Sar , it will be concatenated 
Maybe you forgot about it i guess

Reply:
😂 lol it will not just fix it

Reply:
What you are trying to say with "fix" ?🤔
Everything is fixed with pep8 and deepsource checks

Reply:
you know word "Indentation" ?
ohh if pep8 Deepsource comment on your pr then you will fix until not 😐

Reply:
Ofc , they probably analyse it to check whether the statement is breaking or not
But its not even breaking or not having any indentation error as per PEP guides...

I would prefer you to read PEP guides to ensure your doubt about "indentation"...

Reply:
hmm okay you dont want to understand.

Reply:
> you dont want to understand.

I dont wanna do a useless commit only for you 🤒